frontend/src/components/SyncEditor/SyncDiff.css

```diff
@@ -0,0 +1,38 @@
+.pf-c-code-changes__container .diff-row {
```

Comment:
are these patternfly styles? If so are modifying them globally?

Reply:
yeah, pf-c-code-changes is my class--I changed to sync-diff__container to avoid confusion